backend/docker-compose.yml

```
@@ -0,0 +1,73 @@
+services:
+  api:
+    build:
+      context: .
+      dockerfile: Dockerfile
+    command: uvicorn user_management.main:app --host 0.0.0.0 --port 8000 --reload
+    volumes:
+      - .:/app
+    ports:
+      - "8000:8000"
+    environment:
+      DATABASE_URL : ${DATABASE_URL}
```

YAMLlint (1.35.1)

[error] too many spaces before colon

(colons)